### ISSUE TITLE
Hyperlinks for Epsilon program locations in the terminal

### ISSUE THREAD

**Issue by kolovos** [Arkaedan/vscode-epsilon#1]
It'd be great if the extension could recognise Epsilon program locations in the integrated VS Code terminal (e.g. `/Users/foo/example/program.egl@32:21-32:28` in the stack trace below) and [turn them into clickable hyperlinks](https://stackoverflow.com/questions/60061732/can-we-customize-link-detection-in-vscodes-integrated-terminal).

```
> Configure project :
<=============> 100% CONFIGURING [789ms]
> IDLE
[ant:epsilon.egl - program.egl] EXCEPTION: Property 'foo' not found in object [snip/]
        at (/Users/foo/example/program.egl@32:21-32:28)
        at (/Users/foo/example/program.egl@32:21-32:30)
```

**Comment by Arkaedan** [comment]
Thanks Dimitris! I've just published v1.2.0 to the Visual Studio Marketplace with this addition.

**Comment by kolovos** [comment]
Great! I've updated to 1.2.0 and location detection seems to be working fine on both MacOS and Windows.
